docs/output.md

```diff
@@ -210,7 +210,7 @@ The library-level alignments associated with the same sample are merged and subs
     *Description*:  
     DESeq2 is more commonly used to perform differential expression analysis for RNA-seq datasets. However, it can also be used for ATAC-seq differential accessibility analysis, in which case you can imagine that instead of counts per gene for RNA-seq data we now have counts per accessible region.  
 
-    This pipeline uses a standardised DESeq2 analysis script to get an idea of the reproducibility within the experiment, and to assess the overall differential binding. Please note that this will not suit every experimental design, and if there are other problems with the experiment then it may not work as well as expected.
+    This pipeline uses a standardised DESeq2 analysis script to get an idea of the reproducibility within the experiment, and to assess the overall differential binding. Please note that this will not suit every experimental design, and if there are other problems with the experiment then it may not work as well as expected. For larger experiments, it may be recommended to use the `vst` transformation instead of the default `rlog` option. You can do this by providing the `--deseq2_vst` parameter to the pipeline.
```

Comment:
I would quickly clarify why you recommend it for larger experiments.

Reply:
Good spot with the typo!

Reply:
Easier to link to the docs! Fixed https://github.com/nf-core/atacseq/pull/93/commits/391ae6ad9b277d8f3b26f3c944e2059bcac0c7a5